{{cookiecutter.project_slug}}/clients/mobile/react-native/.gitignore

```diff
@@ -0,0 +1,88 @@
+node_modules/**/*
```

Comment:
hmmm...I would think a file like this would be the same for all projects

Reply:
Not sure what this comment means can you elaborate?